.github/actions/cpUtility-testing/action.yml

```diff
@@ -0,0 +1,63 @@
+name: Build and Push Java-Agent Image after CPUtility Test
+description: |
+  cpUtility Testing
+  This action assumes that Repo was checked out and Java was set correctly
+
+inputs:
+  aws-region:
+    required: true
+    description: "AWS Region"
+  image_uri_with_tag:
+    required: true
+    description: "Image URI with Tag"
+  image_registry:
+    required: true
+    description: "Image Registry"
+  adot-java-version:
+    required: true
+    description: "ADOT Java Version"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        role-to-assume: ${{ secrets.JAVA_INSTRUMENTATION_SNAPSHOT_ECR }}
```

Comment:
Does this role have the correct access to both ecr repos?

Reply:
yes, it has: https://github.com/aws-observability/aws-otel-ops/blob/4859e9eadade7088b4801992344409bbf2fa5b3a/iam-role-setup/lib/config/iam/integ-test-java-config.ts#L28